chain/network/src/peer.rs

```diff
@@ -289,15 +289,15 @@ impl Peer {
                     archival,
                 }) => {
                     let handshake = match act.protocol_version {
-                        37..=PROTOCOL_VERSION => PeerMessage::Handshake(Handshake::new(
+                        39..=PROTOCOL_VERSION => PeerMessage::Handshake(Handshake::new(
```

Comment:
The version used in #3302 was [38](https://github.com/nearprotocol/nearcore/blob/d5f70b339ec5a76e1cc605d30d0fc676bcc9f96e/core/primitives/src/version.rs#L21) not 39.

```suggestion
                        38..=PROTOCOL_VERSION => PeerMessage::Handshake(Handshake::new(
```

Reply:
That is because of this bug https://github.com/nearprotocol/nearcore/pull/3381